[feature] create route for club

RuthP opened the pull request:
# Description

Add club-model
Add club-route

RuthP committed:
[feature] create route for club
Description
========
Add club-model
Add club-route

david-suarez commented on models/club-model.js:
```diff
@@ -0,0 +1,28 @@
+var ClubModel = mongoose.model('Club', ClubSchema);
+module.exports = ClubModel;
+
+
```
It is enough one new line at the end of the file remove the rest please.

david-suarez commented on routes/api-v1.0/club-route.js:
```diff
@@ -0,0 +1,74 @@
+ */
+var express = require('express');
+var route = require('./routes');
+var ClubModel = require('../../models/club-model');
```
Remove this useless variable.

david-suarez commented on routes/api-v1.0/club-route.js:
```diff
@@ -0,0 +1,74 @@
+var route = require('./routes');
+var ClubModel = require('../../models/club-model');
+var Q = require('q');
+var __bind =function(fn, me){ return function(){ return fn.apply(me, arguments); }; };
```
Each line must have 80 characters as maximum. Please read this document:
https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml

david-suarez commented on routes/api-v1.0/club-route.js:
```diff
@@ -0,0 +1,74 @@
+            });
+        //} else {
+          //  response.json('400', {'message': 'Bad Request'});
+        //}
```
If the commented code is useless please remove it.

RuthP committed:
[feature] create route for club
Description
========
Delete spaces
Delete useless variables
Delete the commented code
Reduce size of the variable

david-suarez committed:
[config] Fix configuration of express.

Description
==========
Minor fixes
